medication requests/README.md

```diff
@@ -0,0 +1,308 @@
+| extension\[repeatInformationExtensions].url       | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationRepeatInformation-1"`                                                                                       |
+| extension\[repeatInformationExtensions].Extension | a list of extensions related to `Repeat Information`. Details below in [Repeat Information Extensions](#repeat-information-extensions)                                                                       |
+| extension\[statusChangeExtensions].url            | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/Extension-CareConnect-GPC-MedicationStatusReason-1"`                                                                                            |
+| extension\[statusChangeExtensions].Extension      | a list of extensions related to `Status Change`. Details below in [Status Change Extensions](#status-change-extensions)                                                                                      |
+| status                                            | `"COMPLETED"`<sup>4></sup>, `"STOPPED"`<sup>5</sup> or `"ACTIVE"`<sup>6</sup> <sup>7</sup>                                                                                                                   |
+| note[].text                                       | a list of all values of `ehrSupplyAuthorise / pertinentInformation / pertinentSupplyAnnotation / text` suffixed by a new line<sup>7</sup>                                                                    |
+| priorPrescription.reference                       | contains a reference to a `MedicationRequest` with an id of `ehrSupplyAuthorise / predecessor[0] / priorMedicationRef / id [@root`<sup>8</sup>                                                               |
+| medication.reference                              | contains a reference to a mapped [Medication](../medications/README.md) using an id of `MedicationStatement / consumable / manufacturedMaterial \[@code]` when `medicationStatement / consumable` is present |
+
+1. If there is no `pertinentInformation` which has a `pertinentInformationDosage / text`, then a default value of `"No Information available"` is used
+2. If `ehrSupplyAuthorise / quantity` is not present then `dispenseRequest.quantity` is omitted
+3. For each of the `extension`s, if the value required to set `extension.value` is not present then that extension is not added
+4. When `ehrSupplyDiscontinue` is present and any [Status Change Extensions](#status-change-extensions) are present
+5. When `ehrSupplyDiscontinue` is present and no [Status Change Extensions](#status-change-extensions) are present, or when `ehrSupplyDiscontinue` is not present and `ehrSupplyAuthorise / statusCode / code` equals `"COMPLETE"`
```

Comment:
it should only be STOPPED if the the discontinue and the extension is present

Reply:
are the footnotes the wrong way round?

Reply:
corrected